src/main/java/com/tournament/cup/play/WriterFile.java

```diff
@@ -0,0 +1,23 @@
+package com.tournament.cup.play;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+@Repository
+@AllArgsConstructor
+public class WriterFile {
+    public void writer(String str) {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("results.csv", true));
+            writer.write("\nRound, Team1, Team2, Score");
+            writer.write("\n" + str + "\n");
+            writer.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
This is critical error that should be thrown on top and terminate application. We can't simply proceed when file is not writable